### PR TITLE
Add typings for gravatar

### PR DESCRIPTION
**Pull request by denis-sokolov**

### CODE REVIEW

File: gravatar/gravatar-tests.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/// <reference path="gravatar.d.ts" />
+
```
Comment (horiuchi) [inline gravatar/gravatar-tests.ts]: Please add `import` statement.

Comment (denis-sokolov) [inline gravatar/gravatar-tests.ts]: Added, let’s see if this works.

File: gravatar/gravatar-tests.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+var gravatar = require('gravatar');
```
Comment (horiuchi) [inline gravatar/gravatar-tests.ts]: Please replace `var` to `import`.

Comment (denis-sokolov) [inline gravatar/gravatar-tests.ts]: Done. Why, though? Plenty of modules in the projects use this syntax. If you choose to migrate to the new syntax at some point, you’ll do it en-masse, no?

File: gravatar/gravatar-tests.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+/// <reference path="gravatar.d.ts" />
+
+import gravatar from 'gravatar';
```
Comment (horiuchi) [inline gravatar/gravatar-tests.ts]: Please change to `import gravatar = require('gravatar');` or The gravatar type definition support `default export` of ES6 format.

Comment (denis-sokolov) [inline gravatar/gravatar-tests.ts]: Changed!

**Comment by horiuchi** [comment]
@denis-sokolov Thanks mate!

**Pull request merged**